packages/backend/src/config/config.staging.ts

```diff
@@ -0,0 +1,41 @@
+import { LogLevel } from '@l2beat/common'
+import { bridges, layer2s, tokenList } from '@l2beat/config'
+import { UnixTime } from '@l2beat/types'
+
+import { bridgeToProject, layer2ToProject } from '../model'
+import { Config } from './Config'
+import { getEnv } from './getEnv'
+
+export function getStagingConfig(): Config {
+  const name = 'Backend/Staging'
+  return {
+    name,
```

Comment:
what do you think about spreading the `...getProductionConfig()` here instead of all the configuration being repeated?